global-hooks/openapi/config-values.yaml

```diff
@@ -220,7 +220,7 @@ properties:
               Proxy URL for HTTPS requests.
           noProxy:
             description: |
-              List of no proxy IP and domain entries.
+              List of no proxy IP and domain entries. For wildcard, you can use domain with dot in the beginning, like ".example.com".
```

Comment:
```suggestion
              List of no-proxy IP and domain entries. You can use a domain with a dot prefix as a wildcard, e.g., ".example.com".
```

Reply:
Well, I don't really like the way it sounds. How'bout `For wildcard-domains, use a domain name with a dot prefix, e.g., ".example.com"`?